packages/server/lib/browsers/browser-cri-client.ts

```diff
@@ -265,7 +264,7 @@ export class BrowserCriClient {
         throw new Error(`Could not find url target in browser ${url}. Targets were ${JSON.stringify(targets)}`)
       }
 
-      this.currentlyAttachedTarget = await create(target.targetId, this.onAsynchronousError, this.host, this.port)
+      this.currentlyAttachedTarget = await create(target.targetId, this.onAsynchronousError, this.host, this.port, undefined, this.protocolManager)
```

Comment:
We should update these to accept an object at some point so we don't need to pass `undefined`.

Reply:
[`987f3d3` (#28060)](https://github.com/cypress-io/cypress/pull/28060/commits/987f3d32c7cc87e0163f174f6ba458b577eac597)